Misc/NEWS.d/next/Core_and_Builtins/2024-07-10-03-13-19.gh-issue-121528._Bo7rw.rst

```diff
@@ -0,0 +1,2 @@
+Fix the process crash when load extension when it's compiled with low
```

Comment:
I think this should be reworded, it's not too clear to someone who's not familiar with the exact issue.

```suggestion
Fix crash when loading extension modules that are compiled with an older
```